src/cocotb/runner.py

```diff
@@ -315,17 +313,21 @@ def test(
 
         # When using pytest, use test name as result file name
         pytest_current_test = os.getenv("PYTEST_CURRENT_TEST", "")
```

Comment:
If I were being pedantic I would update this to not provide a default value and update the logic value below to use `None`, which should be a little more precise.

Reply:
I'm all for pedantry.